lib/publishPluginReleaseEvent.js

```diff
@@ -0,0 +1,28 @@
+const axios = require('axios');
+const Program = require('commander');
+const Chalk = require('chalk');
+
+module.exports = async (pluginInfo) => {
+    if (typeof Program.releaseEventURL !== 'undefined') {
```

Comment:
`if (Program.releaseEventURL) {` is more concise and completely fine in this case.

This is a bit too deeply nested for my taste, maybe add an early return instead.

Also, you could put most of the contents of the `try` block (except the output line) in a separate function.